src/components/category_view.js

```diff
@@ -44,10 +44,10 @@ class CategoryView extends Component {
             return <div>This ({category}) isn't a category.</div>
```

Comment:
Super small, but might want to give this a className similar to account so it shows up on the body

Reply:
oh true and just realized my css and other other things werent pushing